src/payment/strategies/paypal-commerce/paypal-commerce-overlay.spec.ts

```diff
@@ -0,0 +1,46 @@
+import OverlayPaypal, { idOverlay } from './paypal-commerce-overlay';
+
+// tslint:disable:no-non-null-assertion
+describe('OverlayPaypal', () => {
+    let overlay: OverlayPaypal;
+
+    beforeEach(() => {
+        overlay = new OverlayPaypal();
+    });
+
+    afterEach(() => {
+        const element = document.getElementById(idOverlay);
+
+        if (element) {
+            element.remove();
+        }
+    });
+
+    it('shows modal element', () => {
+        overlay.show();
+
+        const modal: HTMLElement | null = document.querySelector(`#${idOverlay} .paypal-commerce-overlay_modal`);
```

Comment:
I dont think you need to specify the type here. It can be inferred from what's returned by querySelector()